preprod.healdata.org/portal/gitops.json

```diff
@@ -133,7 +133,10 @@
         { "field": "category", "name": "Study Type"},
         { "field": "clinical_trial_website", "name": "Websites"},
         { "field": "nct_number", "name": "NCT Number"},
-        { "field": "publications", "name": "Study Publications"}
+        { "field": "publications", "name": "Study Publications"},
+        { "field": "priniciple_investigator", "name": "Principle Investigator"},
+        { "field": "publisher", "name": "Administering IC"},
+        { "field": "source", "name": "Research Program"}
```

Comment:
Should the "location" field be here as well?

Reply:
yes!! thanks!

Reply:
oh wait, I thought 'location" will be displayed in the portal also without the field mapping? location sits in rows 123 and 127. hm.. ?

Reply:
you don't need to add it here if you are fine with `Location` as the field name displayed